src/js/otus/queries.tsx

```diff
@@ -61,7 +63,7 @@ export function useInfiniteFindOTUS(refId: string, term: string, verified?: bool
  * @returns A single OTU
  */
 export function useFetchOTU(otuId: string) {
-    return useQuery<OTU, ErrorResponse>(OTUQueryKeys.detail(otuId), () => getOTU(otuId), {
+    return useQuery<OTU, ErrorResponse>(OTUQueryKeys.details(), () => getOTU(otuId), {
```

Comment:
This is likely to cause problems, how does the query tell the difference between request wanting different OTUs when hitting the cache?